Jenkinsfile

```diff
@@ -99,22 +99,17 @@ node() {
             }
           }
         }
-      } catch(err) {
-        echo(err)
-        currentBuild.result = "FAILURE"
-
-        if (err.toString().contains('exit code 143')) {
-          currentBuild.result = "ABORTED"
-        }
```

Comment:
If the error is thrown to the top level, which happens automatically with the `catch` block removed, the build result will be set appropriately. IDK if there was an important reason for the `err.toString().contains('exit code 143')` check that set the build result to `ABORTED`. I deleted it because I don't think it really matters what the build result is if it's not a success, just that it wasn't a success. If there is an important reason for it being there, I can add it back.

Reply:
I don't know either. Looks like that was added in [1952](https://github.com/jenkinsci/blueocean-plugin/pull/1952), when @halkeye added support for building on Sauce Labs.

Reply:
exit code 143 tends to be when a maven is stopped due to another branch of a parallel pipeline execution.  But I don't feel like this adds much value.